e107_handlers/e_marketplace.php

```diff
@@ -492,26 +538,30 @@ class e_marketplace_adapter_wsdl extends e_marketplace_adapter_abstract
 	
 	public function __construct()
 	{
+
+
 		e107_ini_set('soap.wsdl_cache_enabled', 0);
 		e107_ini_set('soap.wsdl_cache_ttl', 0);
 		
 		$options = array(
-			"trace" 				=> true, 
-			'exception' 			=> true,
-		    "uri" 					=> "http://server.soap.e107.inc.com/",
-		    'cache_wsdl'			=> WSDL_CACHE_NONE,
-		    'connection_timeout' 	=> 60,
+			"trace"					=> true, 
+			'exception'				=> true,
+			"uri"					=> "http://server.soap.e107.inc.com/",
+			'cache_wsdl'			=> WSDL_CACHE_NONE,
+			'connection_timeout'	=> 60,
 		);
 

```

Comment:
Your tabs/spaces are showing as changes. Makes it difficult to track what you changed. :-(